docs/theme/src/guide/markdown/include.md

```diff
@@ -300,7 +300,9 @@ pushd "%CD%"
 CD /D "%~dp0"
 ```
 
-@tab C#
+<!-- Half width # cannot be displayed properly -->
```

Comment:
`\#` shoukd work, try that

Reply:
it works

Reply:
The markdown enhance docs should be updated as well